tiatoolbox/data/__init__.py

```diff
@@ -102,10 +104,12 @@ def _local_sample_path(path: pathlib.Path) -> pathlib.Path:
     """Get the path to a data file bundled with the package.
 
     Args:
-        path (pathlib.Path): Relative path to the package data file.
+        path (pathlib.Path):
```

Comment:
Perhaps we should make a top level `types` module for defining our own types such as `PathLike` which is `Union[str, Path]` to avoid having to write out `str or Path` everywhere.

Reply:
Agreed.